src/kakarot/instructions/system_operations.cairo

```diff
@@ -453,6 +456,27 @@ namespace SystemOperations {
             return evm;
         }
 
+        tempvar is_max_depth_reached = 1 - is_not_zero(
```

Comment:
this is forgotten from previous refacto right?

Reply:
yes, added as a scout commit